src/Config.h

```diff
@@ -10,6 +10,7 @@ struct Config
     std::string server_time_zone;
     std::string websso_login_url;
     std::string websso_logout_url;
+	std::string gsc_repo;
```

Comment:
If you look at this code on GitHub you’ll see it’s indented funny. The reason is that there are tab characters in your code—pure evil!—and you have your editor set to display them as 4 spaces. Indenting by 4 is good! But setting tabstops to anything but 8 is bad, because many programs (like github.com) assume 8 ;)

Reply:
Damn! Ok I will change that;